backend/src/appointment/middleware/l10n.py

```diff
@@ -5,6 +5,21 @@
 from ..defines import SUPPORTED_LOCALES, FALLBACK_LOCALE
 
 
+def get_fluent(locales: list[str]):
+    """Provides fluent's format_value function for given locales"""
+
+    # Make sure our fallback locale is always in locales
+    if FALLBACK_LOCALE not in locales:
+        locales.append(FALLBACK_LOCALE)
+
+    base_url = 'src/appointment/l10n'
+
+    loader = FluentResourceLoader(f'{base_url}/{{locale}}')
+    fluent = FluentLocalization(locales, ['main.ftl', 'email.ftl', 'fields.ftl'], loader)
+
+    return fluent.format_value
```

Comment:
I thought it's better to let this live directly in the module, but let me know if you prefer it to be a `@staticmethod`

Reply:
This is fine, it took me a minute to remember how the context system is setup 😅.

Reply:
yeah, me too 😅